task30/src/test/java/by/tut/webpages/myTabel.java

```diff
@@ -0,0 +1,13 @@
+package by.tut.webpages;
+
+public class myTabel {
+    public String name;
+    public String position;
+    public String office;
```

Comment:
Make them private and add setters/getters (standard for model classes)

Reply:
done.